src/v/crypto/CMakeLists.txt

```diff
@@ -1,4 +1,9 @@
-find_package(OpenSSL 3.0.9 EXACT REQUIRED)
+if (RP_OPENSSL_VERSION)
+  message(STATUS "Searching for exactly OpenSSL version ${RP_OPENSSL_VERSION}")
```

Comment:
Do we really need this?

Reply:
i can remove it, was just a nice way of verifying it was working :)